astroquery/skyview/core.py

```diff
@@ -197,15 +199,19 @@ def get_images(self, position, survey, *, coordinates=None, projection=None,
                                                  projection=projection, pixels=pixels, scaling=scaling,
                                                  sampler=sampler, resolver=resolver, deedger=deedger,
                                                  radius=radius, height=height, width=width,
-                                                 cache=cache, show_progress=show_progress)
+                                                 cache=cache, show_progress=show_progress,
+                                                 get_query_payload=get_query_payload)
+        if get_query_payload:
+            return readable_objects
         return [obj.get_fits() for obj in readable_objects]
 
     @prepend_docstr_nosections(get_images.__doc__)
     def get_images_async(self, position, survey, *, coordinates=None,
                          projection=None, pixels=None, scaling=None,
                          sampler=None, resolver=None, deedger=None,
                          radius=None, height=None, width=None,
-                         cache=True, show_progress=True):
+                         cache=True, show_progress=True, get_query_payload=False,
+                         **kwargs):
```

Comment:
again, do not add kwargs
```suggestion
                         cache=True, show_progress=True, get_query_payload=False):
```